SpringProject/src/it/java/com/gradproject2019/UserControllerIT.java

```diff
@@ -81,6 +81,19 @@ public void shouldReturn400WhenAnyFieldNull() throws URISyntaxException {
         Assert.assertEquals(400,response.getStatusCodeValue());
     }
 
+    @Test
+    public void shouldReturn400WhenInvalidPassword() throws URISyntaxException {
```

Comment:
This name is a bit confusing, I thought it was testing the login. I'd rename it to something like shouldReturn400WhenPasswordWrongFormat or ...PasswordInvalidFormat. It might be helpful to rename anything containing 'invalid password' in the rest of the code too.